packages/xchain-cosmos/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@xchainjs/xchain-cosmos",
-  "version": "0.16.1",
+  "version": "0.17.0",
```

Comment:
Suggestions: Let's start with `xchain-cosmos@0.17.0-alpha.1` - so we can add fixes / tweaks easily by increasing `alpha` without bumping the final version.